junit5/src/main/java/org/jboss/weld/junit5/auto/WeldJunit5AutoExtension.java

```diff
@@ -23,7 +23,31 @@
 
 import static org.jboss.weld.junit5.ExtensionContextUtils.getExplicitInjectionInfoFromStore;
 
-public class WeldJunit5AutoExtension extends org.jboss.weld.junit5.WeldJunit5Extension {
+import org.jboss.weld.junit5.WeldJunit5Extension;
+import org.jboss.weld.junit5.WeldJunitEnricher;
+
+/**
+ * An alternative to {@link WeldJunit5Extension} allowing to fully leverage annotation based configuration approach.
+ * When used, the extension will attempt to resolve all beans used in your test class and automatically adds them to
+ * Weld container while bootstrapping it.
+ * 
+ * There is quite a few annotations which can be used to configure it further still:
+ * @see ActivateScopes
+ * @see AddBeanClasses
+ * @see AddEnabledDecorators
+ * @see AddEnabledInterceptors
+ * @see AddExtensions
+ * @see AddPackages
+ * @see EnableAlternatives
+ * @see EnableAlternativeStereotypes
+ *
+ * Note that this approach cannot be combined with manual configuration, e.g. you cannot specify {@code WeldInitiator}
```

Comment:
What happens if I do? Do we throw some exception?

Reply:
> What happens if I do?

I don't know. Undefined I'd say. I know of no way allowing me to detect if other extension is actually enabled in JUnit and I think it unnecessary to scan the class for those combinations.